src/EA.Weee.RequestHandlers/Mappings/ReturnMap.cs

```diff
@@ -53,7 +53,7 @@ public ReturnData Map(ReturnQuarterWindow source)
 
             if (source.ObligatedWeeeSentOnList != null)
             {
-                returnData.ObligatedWeeeSentOnData = source.ObligatedWeeeSentOnList.Select(n => new WeeeObligatedData(n.Id, new Aatf(n.WeeeSentOn.Aatf.Id, n.WeeeSentOn.Aatf.Name, n.WeeeSentOn.Aatf.ApprovalNumber), n.CategoryId, n.NonHouseholdTonnage, n.HouseholdTonnage)).ToList();
+                returnData.ObligatedWeeeSentOnData = source.ObligatedWeeeSentOnList.Select(n => new WeeeObligatedData(n.Id, new Aatf(n.WeeeSentOn.Aatf.Id, n.WeeeSentOn.Aatf.Name, n.WeeeSentOn.Aatf.ApprovalNumber), n.CategoryId, n.NonHouseholdTonnage, n.HouseholdTonnage) { WeeeSentOnId = n.WeeeSentOn.Id }).ToList();
```

Comment:
if this was a bug - can you create a unit test to ensure that the WeeeSentOnId is set